nilearn/maskers/tests/test_multi_nifti_labels_masker.py

```diff
@@ -355,6 +355,41 @@ def test_multi_nifti_labels_masker_resampling():
         )
 
 
+def test_multi_nifti_labels_masker_list_of_sample_mask():
```

Comment:
This is something else than the main topic of this PR (?)

Reply:
Hm that's weird. It's not in the diff of the latest commit. Maybe you looked at the diff of the merge commit. I think if you check the latest you won't see it.